### PR TITLE
Ousman37 patch 1

### PR DESCRIPTION
**Pull request by Ousman37**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/Ousman37.json [hunk 1]
```diff
@@ -1,17 +1,89 @@
 {
   "name": "Ousman Ethmane",
+  "type": "personal",
+  "displayStatsPublic": true,
```
Comment (SaraJaoude) [inline data/Ousman37.json:4]: these are not necessary:
```suggestion
```

File: data/Ousman37.json [hunk 2]
```diff
@@ -1,17 +1,87 @@
 {
   "name": "Ousman Ethmane",
   "bio": "Passionate Front-End Developer | UI/UX Enthusiast | Building innovative web experiences",
+  "avatar": "https://github.com/Ousman37.png",
```
Comment (SaraJaoude) [inline data/Ousman37.json:4]: this is also not necessary: 
```suggestion
  "bio": "Passionate Front-End Developer | UI/UX Enthusiast | Building innovative web experiences",
```

File: data/Ousman37.json [hunk 3]
```diff
@@ -1,17 +1,86 @@
...
+    "Web Development Tools",
+    "Problem Solving",
+    "Debugging",
+    "Team Collaboration"
+  ],
+  "socials": [
+    {
+      "name": "LinkedIn Follow me",
+      "url": "https://www.linkedin.com/in/ethmane-ousman-34885844/",
+      "icon": "fab fa-linkedin",
```
Comment (SaraJaoude) [inline data/Ousman37.json:29]: This is not a valid icon and I have changed this to what I believe you are looking to achieve:
```suggestion
      "icon": "FaLinkedin",
```

**Review by SaraJaoude — DISMISSED** [review]
You do not appear to have used valid icons - please see inline comment and update throughout.  

You may wish to take a look at the [Available Icons page in our Documentation.](https://www.biodrop.io/icons)

File: data/Ousman37.json (hunk 3 above)
Comment (Ousman37) [reply data/Ousman37.json]: Thank you, @SaraJaoude  for your feedback and suggestion. I appreciate your input. I've made the necessary changes to use the custom icon format 'FaLinkedin' as you suggested. Please let me know if there are any further improvements or adjustments needed.

File: data/Ousman37.json [hunk 4]
```diff
@@ -1,17 +1,86 @@
...
+  "evenTags": [
+    "Front-End Development",
+    "UI/UX Design",
+    "Responsive Web Design",
+    "Version Control",
+    "Web Development Tools",
+    "Problem Solving",
+    "Debugging",
+    "Team Collaboration"
+  ],
```
Comment (kumarsonsoff3) [inline data/Ousman37.json:24]: I don't think if `evenTags` is a valid attribute, Please remove this from your `json`. You can add these tags to `tags` attribute if you want. 

```suggestion
```

File: data/Ousman37.json [hunk 5]
```diff
@@ -1,17 +1,86 @@
...
+      "author": "John Doe",
+      "position": "CEO, XYZ Company",
+      "quote": "Ousman is an incredibly talented front-end developer. His work has greatly improved our website's user experience."
+    },
+    {
+      "author": "Jane Smith",
+      "position": "UI/UX Designer, ABC Agency",
+      "quote": "Working with Ousman has been a pleasure. He has a keen eye for design and pays great attention to detail."
+    }
+  ],
```
Comment (kumarsonsoff3) [inline data/Ousman37.json:56]: It doesn't seem to be the right way to add testimonials & may not be displayed on your profile. Please check out [these docs](https://www.biodrop.io/docs/how-to-guides/testimonials-json) to know more about testimonials

```suggestion
```

File: data/Ousman37.json [hunk 6]
```diff
@@ -1,17 +1,86 @@
...
+      "url": "https://twitter.com/Ethamne_dev",
+       "icon": "FaTwitter",
+      "color": "#1DA1F2"
     }
-  ]
+  ],
+  "callToAction": {
+    "text": "Contact Me",
+    "url": "mailto:ousman.didibela@gmail.com"
+  }
```
Comment (kumarsonsoff3) [inline data/Ousman37.json:83]: `callToAction` doesn't seem to be a valid attribute. You can add mail url in the `links` array as a link. 

```suggestion
  ]
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

File: data/Ousman37.json (hunk 4 above)
Comment (Ousman37) [reply data/Ousman37.json]: Updated JSON structure: Removed 'evenTags' attribute and added tags to 'tags' attribute as suggested.

**Comment by SaraJaoude** [comment]
We look forward to seeing your Profile on BioDrop, but as there is no recent activity from you to make the changes suggested I am going to close this pull request.

We hope you will raise a new pull request to create your Profile and when you do, you might want to take a look at t[hese Docs](https://www.biodrop.io/docs/quickstart-json) which will help with the suggested changes the maintainers have proposed. 

Please remember you can also create your Profile using [Forms](https://www.biodrop.io/docs/quickstart-forms).

**Pull request closed without merging**